src/sessions/video-session-handler.ts

```diff
@@ -589,9 +591,13 @@ export class VideoSessionHandler extends BaseSessionHandler {
 
       // if we are unmuting, we need to get a new camera track and add that to the session
     } else {
-      // Make sure we don't have any tracks before we decide to spin up another.
-      if (session._outboundStream.getVideoTracks().length > 0) {
-        this.log('debug', 'Cannot unmute, a video track already exists', { conversationId: session.conversationId, sessionId: session.id, sessionType: session.sessionType });
+      // Check if we already have a camera track (exclude screen share tracks)
+      const existingCameraTrack = session._outboundStream.getVideoTracks().find(track => {
+        return !session._screenShareStream || !session._screenShareStream.getVideoTracks().find(screenTrack => screenTrack.id === track.id);
```

Comment:
It looks like this uses the same logic as line 571. I wonder if we should extract it into its own function. I feel like it'd be easy for me to overlook the need to check for screen share tracks. But I also acknowledge that it's short and we probably won't need to change it anytime soon.